gaugan/callbacks.py

```diff
@@ -0,0 +1,31 @@
+import tensorflow as tf
+from tensorflow.keras import callbacks
+
+from .utils import plot_results
+
+
+class GanMonitor(callbacks.Callback):
```

Comment:
Might be a good idea to serialize the intermediate images rather than plotting them.